public/angular/sass/modals.scss

```diff
@@ -40,3 +48,28 @@
     color: #000;
   }
 }
+
+// Incident modals
+
+.report-to-existing {
+
+  .modal-lg {
+    width: 90%;
+  }
+
+  .modal-body {
+
+    tbody tr {
+      transition: background 0.2s ease-in-out;
+
+      &.item:hover {
+        cursor: pointer;
+        background-color: #eef3f7;
+      }
+    }
+  }
+
+  .modal-footer {
+    margin-top: 0;
+  }
+}
```

Comment:
Looks like most of these styles should be common to all modals, not just incident modal. The issue description says "Modal style should be updated and re-used app-wide." Please consider this. Thanks.